Automattic/jetpack
# CRM Automations UI: Add workflow saving and reloading

Fixes https://github.com/Automattic/zero-bs-crm/issues/3203

## Proposed changes:


This PR implements the saving and reloading of workflows.

### Other information:


## Jetpack product discussion


NtYgueI8oAgiWGHTbHpnjh-fi-1_29

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






Be sure you have test workflows on your site. If not then see this PR for instructions on making some: https://github.com/Automattic/jetpack/pull/33416.

1. Visit `/wp-admin/admin.php?page=jpcrm-automations#/automations`.
2. Toggle the active toggles on some of your workflows. Hard refresh the page to verify that the state is saved.
3. Select some workflows and use the bulk action to change their active state. Hard refresh the page to verify that the state is saved.
4. Click "Edit" for a workflow and change some attributes, then press "Save".  Hard refresh the page to verify that the state is saved.
5. Click "Edit" for a workflow, change some attributes, then press "Cancel". Now press "Edit" again and verify that your changes are gone. Hard refresh the page and click "Edit" again to verify that your changes were not saved.
